src/Proto.Remote/EndpointManager.cs

```diff
@@ -38,7 +41,7 @@ public Task ReceiveAsync(IContext context)
             {
                 case Started _:
                 {
-                    Console.WriteLine("[REMOTING] Started EndpointManager");
+                    _logger.LogDebug("Started EndpointManager");
                     return Actor.Done;

```

Comment:
I actually liked this [REMOTING] tag, as it is useful in tests to see where a log was being written. i.e. in tests where remote nodes are provisioned (even better would be some more info about _which_ remote node)